tests/test_mock.py

```diff
@@ -0,0 +1,58 @@
+from typing import Callable
+from unittest.mock import Mock, create_autospec
+
+from pnpq.apt.connection import AptConnection
+from pnpq.apt.protocol import (
+    Address,
+    AptMessage,
+    AptMessage_MGMSG_MOT_GET_USTATUSUPDATE,
+    AptMessage_MGMSG_MOT_MOVE_ABSOLUTE,
+    ChanIdent,
+    Status,
+    StatusBits,
+)
+from pnpq.devices.polarization_controller_thorlabs_mpc320 import (
+    PolarizationControllerThorlabsMPC320,
+)
+from pnpq.units import pnpq_ureg
+
+
+def test_polarization_controller_move_absolute() -> None:
+
+    connection = create_autospec(AptConnection)
+
+    def mock_send_message_expect_reply(
+        sent_message: AptMessage,
+        match_reply_callback: Callable[
+            [
+                AptMessage,
+            ],
+            bool,
+        ],
+    ) -> None:
+        # Only send a reply if the message is a MOVE_ABSOLUTE message
+        if isinstance(sent_message, AptMessage_MGMSG_MOT_MOVE_ABSOLUTE):
+
+            # A hypothetical reply message from the device
+            reply_message = AptMessage_MGMSG_MOT_GET_USTATUSUPDATE(
+                chan_ident=sent_message.chan_ident,
+                position=sent_message.absolute_distance,
```

Comment:
We need to ask what we're really testing... with this design, we don't have any evidence that the correct values were sent. It would be better if we actually asserted that the `chan_ident` is 1 and the position is as expected on the `sent_message` before using those values to construct the `reply_message`.